pkg/cmd/opts/install.go

```diff
@@ -127,7 +127,8 @@ func (o *CommonOptions) DoInstallMissingDependencies(install []string) error {
 		case "kvm2":
 			err = kvm2.InstallKvm2()
 		case "ksync":
-			_, err = ksync.InstallKSync()
+			sy := &ksync.KsynClier{}
```

Comment:
I find this odd, why would you pass an empty KsynClier struct here. What do you want to achieve?

Reply:
Refactored this code, pretty similar to the existing codebase now.